src/FsCheck/Gen.fs

```diff
@@ -382,9 +382,11 @@ module Gen =
     //[category: Creating generators from generators]
     [<CompiledName("ListOf")>]
     let listOf gn =
-        sized <| fun n ->
-            gen {   let! k = choose (0,n+1) //decrease chance of empty list
```

Comment:
>choose (0, n+1) // decrease chance of empty list

I have a feeling that this shouldn't be the case anymore in the [replace-random](https://github.com/fscheck/FsCheck/tree/replace-random) branch.

In that branch the Hugs/Random's [StdGen](https://hackage.haskell.org/package/random-1.1/docs/System-Random.html) algorithm is replaced by [SplitMix](https://github.com/nkartashov/SplitMix/tree/master/src/System/Random).